packages/react-core/src/layouts/Bullseye/Bullseye.tsx

```diff
@@ -8,7 +8,7 @@ export interface BullseyeProps extends React.HTMLProps<HTMLDivElement> {
   /** additional classes added to the Bullseye layout */
   className?: string;
   /** Sets the base component to render. defaults to div */
-  component?: keyof JSX.IntrinsicElements;
+  component?: React.ReactNode;
```

Comment:
I don't think this is valid, strings that are rendered as tag should be one of the IntrinsicElements only, ReactNode can be `ReactElement | ReactText | ReactFragment | ReactPortal | boolean | null | undefined`